bin/index.js

```diff
@@ -49,7 +51,7 @@ try {
   main = require(path.join(__dirname, '..', 'commands', `${command}.js`))
 } catch (err) {
   console.error(chalk.red(`
-Unknown command: ${command}`))
+Unknown command: ${command}`, err))
```

Comment:
Ahh crap, didnt mean to check this in.

I do like the new flag idea, would you prefer `--debug` over `--verbose`